adsingestp/parsers/dubcore.py

```diff
@@ -53,6 +53,12 @@ def _parse_ids(self):
         self.base_metadata["ids"] = {}
         self.base_metadata["ids"]["pub-id"] = []
 
+        if self.input_header.find("identifier"):
+            for dc_id in self.input_header.find_all("identifier"):
+                self.base_metadata["ids"]["pub-id"].append(
+                    {"attribute": "publisher-id", "Identifier": dc_id.get_text()}
```

Comment:
I talked to the curators, and we'd like to use `urn` as the attribute rather than `publisher-id`.  Would you update this (and the unit test output) accordingly?